src/render/filters.rs

```diff
@@ -274,6 +275,51 @@ impl ResolveFilter for DefaultIfNoneFilter {
     }
 }
 
+impl ResolveFilter for DateFilter {
+    fn resolve<'t, 'py>(
+        &self,
+        variable: Option<Content<'t, 'py>>,
+        py: Python<'py>,
+        template: TemplateString<'t>,
+        context: &mut Context,
+    ) -> ResolveResult<'t, 'py> {
+        let Some(value) = variable else {
+            return Ok(Some("".as_content()));
+        };
+
+        let fmt = match &self.argument {
+            Some(arg) => arg
+                .resolve(py, template, context, ResolveFailures::Raise)?
+                .expect("missing date format argument")
+                .to_py(py),
+            None => {
+                let formats_mod = py.import("django.utils.formats")?;
+                formats_mod.call_method1("get_format", ("DATE_FORMAT",))?
```

Comment:
Instead of using `py.import` we should use the [more efficient approach with `PyOnceLock`](https://pyo3.rs/main/doc/pyo3/sync/struct.pyoncelock#method.import).

See also #244.